### ISSUE TITLE
Fedora 37-1.7 Silverblue DVD install failing with Evince flatpak error

### ISSUE THREAD

**Issue by azzamsa** [fedora-silverblue/issue-tracker#432]
**Describe the bug**
The DVD installations always fail with the error below:

```
The following error occurred while installing the payload. This is a fatal error and installation will be aborted.
Failed to install flatpaks: flatpak-error-quark: Aborted due to failure (While pulling runtime/org.fedoraproject.Platform/x86_64/f37 from remote Anaconda: Writing content object: Error reading from file descriptor: Input/output error) (4)
Exit installer
```

**To Reproduce**
Please describe the steps needed to reproduce the bug:
1. Download the ISO from https://mirror.yer.az/fedora/releases/37/Silverblue/x86_64/iso/Fedora-Silverblue-ostree-x86_64-37-1.7.iso
2. Put it in Ventoy https://www.ventoy.net/en/index.html USB
3. Start the installation

**Expected behavior**
The installation complete

**Screenshots**

![20230316_054608](https://user-images.githubusercontent.com/17734314/225462562-580c1ab6-abfa-405a-a590-5fc4cbf72977.jpg)
![20230316_054608-1](https://user-images.githubusercontent.com/17734314/225462570-80e3791d-f500-4de1-8d69-bcab96ed13cc.jpg)

**Additional context**
Am I need to be connected to the internet during installation?

Related:
- https://github.com/fedora-silverblue/issue-tracker/issues/38

**Comment by azzamsa** [comment]
Is there any hot solution? 

I am tried of my main machine breaks, and I want to migrate to Silverblue. I can't do any productive work before any OS installed in it.

Should I use `Fedora-Silverblue-ostree-x86_64-Rawhide-20230315.n.1.iso`?

**Comment by azzamsa** [comment]
I have checked the file's integrity after downloading. Because sometimes I got an unstable internet connection.

Turns out I need to make sure the file is not corrupted after copying to a flash drive. I never thought this is the cause. It is the first time in my life.

I suspect this was caused by a bug in Nautilus. The shasum keeps failing. I use `cp` from Terminal as a workaround.

Credit:
- Suggested by @no-clip:matrix.org in https://matrix.to/#/#silverblue:fedoraproject.org
